### PR TITLE
Beta2ga pdf ocr

### PR DESCRIPTION
**Pull request by happyhuman**
Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [ ] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by happyhuman** [comment]
Hi @alexander-fenster @jmdobry , @ace-n , @JustinBeckwith Can you please review this PR? Please note that in order to pass the CircleCi tests, I had to change config.yml a little (with some help from Chris Frater).

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,24 +3,32 @@ workflows:
   version: 2
   tests:
     jobs: &workflow_jobs
-      - node6:
+      - node4:
```
Comment (JustinBeckwith) [inline .circleci/config.yml]: Why are we adding node 4 and 9 back to the test config?  Looks like you may need to rebase.

File: samples/detect.js [hunk 2]
```diff
@@ -770,6 +770,71 @@ function detectFulltextGCS(bucketName, fileName) {
...
+      },
+    ],
+  };
+
+  client
+    .asyncBatchAnnotateFiles(request)
+    .then(results => {
+      const operation = results[0];
+      // Get a Promise representation of the final result of the job
+      operation
```
Comment (JustinBeckwith) [inline samples/detect.js]: You don't need to nest your promises :) . Just return `operation.promise()` and use promise chaining to keep things more flat:

```js
.then(results => {
  const operation = results[0];
  return operation.promise();
}).then(filesResponse => {
  // do work
}).catch(console.error);
```

**Comment by happyhuman** [comment]
@JustinBeckwith thanks for your comments. I updated the code per your suggestions.

File: samples/detect.js [hunk 3]
```diff
@@ -770,6 +770,67 @@ function detectFulltextGCS(bucketName, fileName) {
...
+      const operation = results[0];
+      // Get a Promise representation of the final result of the job
+      return operation.promise();
+    })
+    .then(filesResponse => {
+      let destinationUri =
+        filesResponse[0].responses[0].outputConfig.gcsDestination.uri;
+      console.log('Json saved to: ' + destinationUri);
+    })
+    .catch(function(error) {
```
Comment (JustinBeckwith) [inline samples/detect.js]: This all looks so much better!   Little nit here - you don't have to create a wrapping function for the console.log.  You can just do this:

```js
.catch(console.error);
```

**Pull request merged**